WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel+EditProductsM3Tests.swift

```diff
@@ -0,0 +1,173 @@
+import XCTest
+
+@testable import WooCommerce
+@testable import Yosemite
+
+/// The same tests as `DefaultProductFormTableViewModel_EditProductsM2Tests`, but with Edit Products M2 and M3 feature flag on.
+/// When we fully launch Edit Products M2 and M3, we can replace `DefaultProductFormTableViewModel_EditProductsM2Tests` with the test cases here.
+///
+final class DefaultProductFormTableViewModel_EditProductsM3Tests: XCTestCase {
+
+    private let mockFeatureFlagService = MockFeatureFlagService(isEditProductsRelease2On: true, isEditProductsRelease3On: true)
+
+    func testViewModelForSimplePhysicalProductWithoutImagesWhenM3FeatureFlagIsOff() {
```

Comment:
sorry I think this is from a mistake in the [M2 tests](https://github.com/woocommerce/woocommerce-ios/blob/develop/WooCommerce/WooCommerceTests/ViewRelated/Products/Edit%20Product/DefaultProductFormTableViewModel+EditProductsM2Tests.swift#L13), where the M3 feature flag isn't off as described in the test function name (`*WhenM3FeatureFlagIsOff`). I think `WhenM3FeatureFlagIsOff` can just be removed from this function name?

Reply:
fixed https://github.com/woocommerce/woocommerce-ios/pull/2043/commits/67d73521ae6e6696ac8fa809f25d8d2d75e3a3a8